src/Files.App/Data/Items/IniSectionDataItem.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Data.Items
+{
+	/// <summary>
+	/// Represents item of ini file's section.
+	/// </summary>
+	public class IniSectionDataItem
+	{
+		public string SectionName { get; set; } = "";
+
+		public Dictionary<string, string>? Parameters { get; set; }
```

Comment:
```suggestion
		public IDictionary<string, string> Parameters { get; set; } = ImmutableDictionary<string, string>.Empty;
```
This way, there is no need to consider the case where Parameters is null.